example/src/client.ts

```diff
@@ -66,7 +67,7 @@ function createLanguageClient(connection: MessageConnection): BaseLanguageClient
 
 function createUrl(path: string): string {
     const protocol = location.protocol === 'https:' ? 'wss' : 'ws';
-    return `${protocol}://${location.host}${path}`;
+    return normalizeUrl(`${protocol}://${location.host}${location.pathname}${path}`);
```

Comment:
why appending `location.pathname` now?

Reply:
I am accessing the example server deployed on the remote machine and this machine exposing the server on the subpath instead of on the host.